manifests/chew-datasets.toml

```diff
@@ -37,23 +37,40 @@ name = "ipfs-add-defaults"
 instances = { min = 1, max = 1, default = 1 }
 
   [testcases.params]
-  some_param = { type = "int", desc = "some param", unit = "peers" }
+    [testcases.params.file-sizes]
+      type = "string array"
+      desc = "An array of File Sizes to be tested"
+      unit = "SI Units"
+      default = ["1MB", "10MB", "100MB"]
```

Comment:
I think this is a TOML array, but below (Δ) you parse this as JSON?

Reply:
See reply below.